### DISCUSSION TITLE
Visualise only the segmented parts of the image

### DISCUSSION THREAD

**Discussion by jashshah** [facebookresearch/detectron2#4632] [Q&A]
Hello, I just started using detectron2, and was wondering if there is a way using `Visualizer` to only visualise the parts of image that is the result of some segmentation and black/dark everywhere else?

For example from the image below, I'd only like the parts of the image that are segmented i.e. Person 1 to Person 6, the motorcycle and the chair.

![image](https://user-images.githubusercontent.com/7275517/199002342-2a1b299f-590d-4b3a-bad6-78873dabc3ab.png)

**Comment by suryasid09** [comment]
Hello Jash,

I am also new to detectron2. Could you let me know if you are using the [detectron2](https://github.com/facebookresearch/detectron2)/[tools](https://github.com/facebookresearch/detectron2/tree/main/tools)/visualize_data.py to visualise your data? I would really appreciate if you could tell how did you register your data for visualizer?
